supabase/tools/createStorage.ts

```diff
@@ -23,21 +23,24 @@ async function createPublicBucket(supabase: SupabaseClient, name: string): Promi
     });
 
     if (error) {
-        throw new Error(`Failed to create bucket called ${name}, error: ${error}`);
+        throw new Error(`Failed to create bucket called ${name}, error: ${error.message}`);
```

Comment:
I assume this was just wrong before?

Reply:
yup yup the error message came out as `[Object object]` so showing the message instead now